### ISSUE TITLE
[FR]: Better way to add new icons

### ISSUE THREAD

**Issue by luissilva1044894** [israpps/HDL-Batch-installer#40]
### Explanation

Make a new repo and add me as Contributor? Or I should make a [public repo](https://github.com/luissilva1044894/Icon-pack-for-HDL-Batch-installer) which you'll "fork" later?

I'm planning to add as many icons as possible. If you've any idea, tell me.

### fature related

- [ ] game Installation
- [X] Title Database
- [ ] ART Downloader
- [ ] Game management
- [ ] HDD management
- [ ] Others...

### other checks

- [ ] I've checked if such feature exists on [Latest version](https://github.com/israpps/HDL-Batch-installer/releases/)
- [ ] This Feature requires an update of WxWidgets
- [ ] Needs an update of [HDL Dump](https://github.com/ps2homebrew/hdl-dump)

**Comment by israpps** [comment]
Not really,
I'm not sure if it's safe to hold the bare icons on the repo

You can make a repo to organize yourself

And when you want to make the PR just pack the files and do the PR

**Comment by luissilva1044894** [comment]
Why? Copyright or something like? 

Well, isn't that hard to unzip your file... Password is hardcoded in the repo (That's why I got it without asking for). I don't see why it should be less safe.

Whatever, I was thinking something like your FMCB repo automatically uploading files. 
But I think should be better to use _.json_ or _folder_ based system. It's turning hard to follow that _.ini_ (2469 lines currently, gamename.DB has +15000 lines. Turned it alphabetically, but still...) and 7z file will be ever more big.

**Comment by israpps** [comment]
Maybe

But keeping the list as an INI is easier and simpler from code side

Maybe make a system that builds the package from the style you want into the style used by the program

**Comment by luissilva1044894** [comment]
Check my [repo](https://github.com/luissilva1044894/Icon-pack-for-HDL-Batch-installer) now.

It's already generating .7z (With your pass!). Better to review, less prone to errors.

Only changes I really made was: Removed unecessary `#` and reordered `Icons.ini` to match [gamename DB](https://github.com/israpps/HDL-Batch-installer/blob/main/Database/gamename.csv).

**Comment by israpps** [comment]
@luissilva1044894 users are reporting duplicated entries on the ini

Maybe you can check it.

I'm too busy ATM

**Comment by luissilva1044894** [comment]
Well, I'm working on a CDN like for PS2 icons.
There are only 940 icons there rn. Unlike your .zip, some icons are duplicated.

[https://playstation2-icons.onrender.com/{game_id}/icon.sys](https://playstation2-icons.onrender.com/SLUS_207.05/icon.sys) (HDD format. All values extracted from save)
[https://playstation2-icons.onrender.com/{game_id}/list.ico](https://playstation2-icons.onrender.com/SLUS_207.05/list.ico)
[https://playstation2-icons.onrender.com/{game_id}/del.ico](https://playstation2-icons.onrender.com/SLUS_207.05/del.ico) (md5 != list.ico or 404 error)
[https://playstation2-icons.onrender.com/{game_id}/copy.ico](https://playstation2-icons.onrender.com/SLUS_207.05/copy.ico) (md5 != list.ico or 404 error, not needed for HDDOSD)

Or [https://playstation2-icons.onrender.com/{game_id}](https://playstation2-icons.onrender.com/SLUS_207.05) which returns a zip containing all files.

I'm still working on title0 & title1. Looking for a better DB for parsing the names, I'll use SerialStation, I guess.

I'll asking later on #savedata-sharing/GDX, but what u think? Can you send me the saves u have?
